Source/Website/TailwindTraders.Rewards.Website.csproj

```diff
@@ -98,6 +99,13 @@
     <Content Include="Content\fonts\BrandonGrotesque-Light.woff2" />
     <Content Include="Content\fonts\BrandonGrotesque-Medium.woff" />
     <Content Include="Content\fonts\BrandonGrotesque-Medium.woff2" />
+    <None Include="Dockerfile" />
+    <None Include=".dockerignore">
+      <DependentUpon>Dockerfile</DependentUpon>
+    </None>
+    <None Include="Properties\PublishProfiles\FolderProfile.pubxml" />
+    <None Include="Properties\PublishProfiles\TailwindTradersRewardsWebsite20190621084120 - Web Deploy.pubxml" />

```

Comment:
Can we remove the publishing profiles references?

Reply:
Only the Web profile one. The publish to Folder's profile is needed for the dockerfile's correct behaviour.

Reply:
I've just deleted the others.